components/studio/studio/urls.py

```diff
@@ -28,7 +28,7 @@
     path('', views.home, name='home'),
     path('privacy/',views.privacy, name='privacy'),
     path('guide-get_account/', views.partial_render_guide("guide_get_account.html"), name='guide_get_account'),
```

Comment:
I see that it was my mistake, fixed in my branch

Reply:
No, i just changed the name because in the text it refers to "Projects" and there were two pages with the same headline.